### PR TITLE
Macedonian locale

### PR DESCRIPTION
**Pull request by van100j**
* Adds Macedonian locale

### CODE REVIEW

**Comment by Marak** [comment]
Thank you for your contribution @van100j 

At first glance, this looks good. I don't speak Macedonian though.

Can anyone else who speaks Macedonian confirm this Pull Request?

**Review by igorhristov — APPROVED** [review]
I am native Macedonian and confirm this Pull Request.

File: lib/locales/mk/name/male_first_name.js [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+  "Владимир",
+  "Владо",
+  "Властимир",
+  "Воислав",
+  "Војдан",
+  "Војкан",
+  "Волче",
+  "Гаврил",
+  "Гане",
+  "Георги",
```
Comment (igorhristov) [inline lib/locales/mk/name/male_first_name.js:62]: Please add Георге

**Review by igorhristov — APPROVED** [review]

**Comment by Marak** [comment]
Thank you for your contributions @van100j and @igorhristov 

Merged.

**Pull request merged**